src/packages/BalancedJs/contracts/sICX.ts

```diff
@@ -103,4 +103,31 @@ export default class sICX extends Contract {
 
     return this.call(callParams);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
```

Comment:
I'd like to use the names of the same parameters as names of smart contract method params.
toAddress -> to